app/src/main/res/layout/fragment_appcoints_info.xml

```diff
@@ -27,7 +27,7 @@
       android:layout_height="wrap_content"
       android:layout_above="@id/app_cardview"
       android:layout_below="@id/app_bar_layout"
-      android:layout_marginBottom="28dp"
+      android:layout_marginBottom="8dp"
```

Comment:
I think we should remove this margin to be the same as the design mockup. With it it looks like that the top margin of the install card is bigger than the bottom one (because of the same background colour). I have tried without it and I think it looks closer to what we want.